CIP-XXXX/README.md

```diff
@@ -0,0 +1,408 @@
+---
+CIP: ???
+Title: Cardano dApp-MutlisigWallet Web Bridge
+Status: Proposed
+Category: Wallets
+Authors: 
+  - Leo
+Implementors:
+  - BroClan <https://broclan.io/>
```

Comment:
```suggestion
Implementors: NA
```
It might not be necessary to remove BroClan from the list of `Implementors` (which should list only the full implementations at the time of CIP submission) if you have a working implementation in your wallet.

If you do have a complete implementation, then it should be clearly marked as implemented down in your _Implementation Plan_ (currently that box isn't checked).